packages/libraries/client/README.md

```diff
@@ -304,3 +304,30 @@ const server = new ApolloServer({
   ]
 })
 ```
+
+## Self-Hosting
+
+To align the client with your own instance of GraphQL Hive, you should use `selfHosting` options in the client configuration.
+
+The example is based on GraphQL Yoga, but the same configuration applies to Apollo Server and others.
+
+```ts
+import { createYoga } from '@graphql-yoga/node'
+import { useHive } from '@graphql-hive/client'
+
+const server = createYoga({
+  plugins: [
+    useHive({
+      enabled: true,
+      token: 'YOUR-TOKEN',
+      selfHosting: {
+        graphqlEndpoint: 'https://your-own-graphql-hive.com/graphql',
+        applicationUrl: 'https://your-own-graphql-hive.com',
```

Comment:
Nice, this `applicationUrl` I think was the only missing parameter so we can configure everything for `selfHosting` 🎉